velox/functions/prestosql/aggregates/tests/ArrayAggTest.cpp

```diff
@@ -34,6 +35,13 @@ class ArrayAggTest : public AggregationTestBase {
     AggregationTestBase::SetUp();
     registerSimpleArrayAggAggregate("simple_array_agg");
   }
+
+  RowVectorPtr fuzzFlatBatch(const RowTypePtr& rowType) {
```

Comment:
nit: perhaps, fuzzFlat and add a parameter for size

Reply:
Fxied